### ISSUE TITLE
`manage.py flavormessages` command should not generate po files for other flavors

### ISSUE THREAD

**Issue by LukeSwart** [mapseed/platform#536]
Background: We are using Django's i18n feature to generating .po files, which are used for providing translations.

Problem: We are currently generating .po files for all flavors when we run `manage.py flavormessages`. This command should only generate .po files for the specific flavor in our shareabouts settings. It seems like the fix should be fairly straightforward, and done in this file: https://github.com/smartercleanup/platform/blob/master/src/sa_web/management/commands/flavormessages.py

Basically, we should search for the currently configured flavor, and only generate the .po files for that flavor. We should still generate the .po file for our base flavor.

**Comment by goldpbear** [comment]
Closing-- this is addressed in #809
